backend/user_management/minio/schemas.py

```
@@ -0,0 +1,12 @@
+
+
+from pydantic import BaseModel
+from typing import List
+
+class ProductPhotoUploadResponse(BaseModel):
+    message: str
+    urls: List[str]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)